docs/docs-content/clusters/edge/local-ui/host-management/custom-link.md

```diff
@@ -64,31 +64,37 @@ or open a new tab in your browser depending on how to configure the links.
    git checkout v4.4.0
    ```
 
-4. Create a directory called **ui** at the path `CanvOS/ui`.
+4. Create a directory called **local-ui** at the path **CanvOS/local-ui**. Then under the **local-ui** directory, create
```

Comment:
```suggestion
4. Create a directory called **local-ui** at the path **CanvOS/local-ui**. Then, under the **local-ui** directory, create
```

Reply:
For this step, I would maybe rephrase it a bit differently and give a mkdir command? As in:

> Use the following command to create the **local-ui/ui** directory:
> ```
> mkdir CanvOS/local-ui/ui
> ```

Feels a bit more straightforward then splitting it up into multiple dirs 🤔

Reply:
Does this work? I don't think you can create a subdirectory when its parent doesn't exist? But maybe there is a way to override..

```
mkdir CanvOS/local-ui/ui
mkdir: CanvOS/local-ui: No such file or directory
```